.eslintrc.js

```diff
@@ -26,4 +26,10 @@ module.exports = {
     "node/prefer-promises/dns": "error",
     "node/prefer-promises/fs": "error",
   },
+  "prettier/prettier": [
```

Comment:
move this line of code: `"prettier/prettier": [
    "error",
    {
      endOfLine: "auto",
    },
  ],`

and replace it in place of `"prettier/prettier": "error",` inside rules object

e.g: the result should be like this
```
rules: {
    "prettier/prettier": [
        "error",
        {
             endOfLine: "auto",
         },
    ],
    .....
}
```

Reply:
merging this, for now, resolve this from the development branch.